### DISCUSSION TITLE
Rouge TV

### DISCUSSION THREAD

**Discussion by jkemp814** [iptv-org/community#252] [Q&A] [answered]
Does Rouge TV show adult content? I was going through the XXX list and notice the channel showing Baja racing from Red Bull.

**Comment by Dum4G** [comment]
It shows porn during late overnight by European time. I checked

**Reply by BellezaEmporium** [reply, marked as answer]
Yes, which is why I preferred classifying it into the XXX section, even if the day program is clean.

As an example, for today's programmation : 

![image](https://user-images.githubusercontent.com/30985701/122597940-671e0c80-d06c-11eb-8b6f-4461d4e350e9.png)
